src/CopyToClipboard/CopyToClipboard.component.tsx

```diff
@@ -74,7 +74,7 @@ const SCopyToClipboard = styled.div`
   display: ${(props: Props) => (props.fullWidth ? 'flex' : 'inline-flex')};
   padding: ${(props: Props) => props.theme.copyToClipboard.padding};
   background-color: ${(props: Props) =>
-    props.theme.colors[props.background!] || props.background!.toString()};
+    props.background!.toString() || props.theme.colors[props.background!]};
```

Comment:
@ben-dalton @tdakhla From how I understand this new props order, if user pass the `background-color` as `blue`, the component will evaluate the browser's default blue color before evaluating for the theme-defined blue, and hence always fallback to the browser's blue color. Is this the desired outcome?

Reply:
@Dantiff Correct, the incoming prop (if provided) should always override the default theme value.